datev_export_xml/tests/test_datev_export.py

```diff
@@ -81,6 +79,7 @@ def setUp(self):
         self.refund_date = self.today - timedelta(days=55)
         self.start_date = self.today - timedelta(days=34)
         self.end_date = self.today - timedelta(days=32)
+        self.InvoiceObj.search([("state", "=", "posted")]).button_draft()
```

Comment:
Why is it necessary?

Reply:
This is necessary, as there is a restriction in account move that you cannot delete an item linked to a posted entry.